### PR TITLE
Remove all "www"s

### PR DESCRIPTION
**Pull request by leo**
They’re not needed, so there’s no point in keeping them! :blush:

### CODE REVIEW

**Comment by stefanpenner** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ version:
 
 - Users
   + [`ember new` diff](https://github.com/kellyselden/ember-cli-output/compare/v2.4.1...v2.4.2)
-  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + Upgrade your project's ember-cli version - [docs](http://ember-cli.com/#project-update)
```
Comment (rwjblue) [inline CHANGELOG.md]: This should be `http://ember-cli.com/user-guide/#upgrading` (we changed the URL's a while back, but apparently didn't update these links).

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ version:
 
 - Users
   + [`ember new` diff](https://github.com/kellyselden/ember-cli-output/compare/v2.4.0...v2.4.1)
-  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + Upgrade your project's ember-cli version - [docs](http://ember-cli.com/#project-update)
```
Comment (rwjblue) [inline CHANGELOG.md]: Same as above

**Comment by Turbo87** [comment]
@homu r+

**Pull request merged**